README.rst

```diff
@@ -173,13 +173,20 @@ Developing Renku
 ================
 
 For development it's convenient to install ``renku`` in editable mode. This is
-still possible with ``pipx``. First clone the repository and then do:
+still possible with ``pipx``. First clone the repository and run the setup:
+
+::
+
+    $ python setup.py bdist_wheel
+
+Then install with ``pipx`` in editable mode:
 
 ::
 
     $ pipx install \
         --editable \
-        <path-to-renku-python>[all] \
+        --force \
+        --spec <path-to-renku-python> \
```

Comment:
This is no longer the current `pipx` syntax - if you update your pipx the original instructions will work.

Reply:
Oh my bad, I tought I installed the latest version but I was getting it from an outdated source.